hack/make.sh

```diff
@@ -170,10 +170,13 @@ BUILDFLAGS=( $BUILDFLAGS "${ORIG_BUILDFLAGS[@]}" )
 
 if [ "${DOCKER_ENGINE_GOARCH}" == "arm" ]; then
 	: ${TIMEOUT:=210m}
+	: ${CHECKTIMEOUT:=10m}
 elif [ "${DOCKER_ENGINE_GOARCH}" == "windows" ]; then
 	: ${TIMEOUT:=180m}
+	: ${CHECKTIMEOUT:=8m}
 else
 	: ${TIMEOUT:=120m}
+	: ${CHECKTIMEOUT:=5m}
 fi
 
 TESTFLAGS+=" -test.timeout=${TIMEOUT}"
```

Comment:
Would like to remove this, and add `-test.timout=${TIMEOUT}` manually to hack/make/test-unit. That way only unit tests use this. This would also allow for removing the CHECKTIMEOUT variable, and simply using the TIMEOUT for what it shoudl be used: per-test timeout. It's just that unit tests and integration tests are implemented differently.